lib/Language/ja/translation.json

```diff
@@ -1,31 +1,31 @@
 {
   "tour": {
     "exploreMapDataButton": {
-      "content": "## データを地図上で見る\n\nここでは、データカタログからデータを探し、地図に追加することができます。複数のデータも追加できます。追加したデータは、画面左側にリストとして表示されます。"
+      "content": "## データを見る\n\nここでは、データカタログからデータを探し、地図に追加することができます。複数のデータも追加できます。追加したデータは、ワークベンチ内にリストとして表示されます。"
     },
     "locationSearchInput": {
-      "content": "## 場所を検索\n\n地名や住所を入力するとその場所を地図上で検索できます。"
+      "content": "## 場所検索\n\n地図上の興味ある場所の名前や住所を入力してください。"
```

Comment:
How do you know which of these to keep?

Reply:
Good catch!

The `master` branch don't have these translations, so these changes can't be from master. I suspect the changes are due to some encoding differences in my local machine. I have reverted the whole file to previous version.